Makefile

```diff
@@ -15,7 +15,7 @@ ifdef E2E_TESTS_MODE
 E2E_TESTS_CONFIG = --img-expr-time=5m --img-expr-interval=5m
 endif
 
-ifeq ($(CONTAINER_COMMAND), podman)
+ifneq (,$(findstring podman,$(CONTAINER_COMMAND)))
```

Comment:
Does that mean that `podman` is chosen if both `docker` and `podman` exist on the machine?

Reply:
no,  this section is for local registry and tells podman to push even if the registry communicates over http (unsecured).
We have two different podman tools: podman for the host and podman-remote (can use podman --remote but uses podman-remote command for readability) for the skipper container.
This condition checks if the CONTAINER_COMMAND contains the word "podman" so for docker the value will be false